### PR TITLE
Modified TOC, corrected bad link

### PR DESCRIPTION
**Pull request by rpetrusha**
# Modified TOC, corrected bad link

## Summary

Added getting started tutorials to the .NET Core Getting Started section

## Suggested Reviewers

@mairaw

### CODE REVIEW

File: docs/toc.md [hunk 1]
```diff
@@ -390,6 +395,8 @@
 # [Visual Basic Guide](visual-basic/index.md)
 ## [Get Started](visual-basic/getting-started/index.md)
 ### [What's New for Visual Basic](visual-basic/getting-started/whats-new.md)
+### [Build a Visual Basic Hello World app with .NET Core in Visual Studio 2017](core/tutorials/w)
```
Comment (mairaw) [inline docs/toc.md]: broken link

File: docs/toc.md [hunk 2]
```diff
@@ -390,6 +395,8 @@
...
+### [Build a class library with Visual Basic and .NET Core in Visual Studio 2017](core/tutorials/vb-library-with-visual-studio.md)
```
Comment (mairaw) [inline docs/toc.md]: why do we have the same link in two places? For example, you'll end up with two locations on the TOC highlighted and screen will always sync on the first one, even if you click inside the VB toc. e.g. https://review.docs.microsoft.com/en-us/dotnet/core/tutorials/vb-library-with-visual-studio?branch=pr-en-us-2916

File: docs/toc.md [hunk 3]
```diff
@@ -55,6 +55,11 @@
 <!-- .NET Core Content -->
 # [.NET Core Guide](core/index.md)
 ## [Get started](core/get-started.md)
+### [Get started with C# and Visual Studio Code](core/tutorials/with-visual-studio-code.md)
```
Comment (mairaw) [inline docs/toc.md]: I was actually thinking about these topics this week and I think they belong under Get Started as you did here. Let's test this new location and if we feel that's the right spot, we should also move physical locations (again).

**Review by mairaw — CHANGES_REQUESTED** [review]

File: docs/toc.md (hunk 2 above)
Comment (rpetrusha) [inline docs/toc.md]: @mairaw, I've removed it.

**Comment by rpetrusha** [comment]
@mairaw, addressed your comments and removed the duplication from the TOC.

**Comment by rpetrusha** [comment]
Closing and reopening to restart build.

File: docs/toc.md [hunk 4]
```diff
@@ -55,6 +55,11 @@
...
+### [Build a C# Hello World app with .NET Core in Visual Studio 2017](core/tutorials/with-visual-studio.md)
+### [Build a Visual Basic Hello World app with .NET Core in Visual Studio 2017](core/tutorials/w)
```
Comment (mairaw) [inline docs/toc.md]: broken link

**Review by mairaw — APPROVED** [review]
LGTM but build report still showing warnings. I've highlighted the one I found. Good to merge once builds are green.

**Pull request merged**